apps/services/sessions/project.json

```diff
@@ -4,6 +4,22 @@
   "projectType": "application",
   "targets": {
     "build": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "outputPath": "dist/apps/services/sessions",
+        "commands": ["nx esbuild services-sessions"],
+        "parallel": false
```

Comment:
These should be fine to run in parallel ... right?

Reply:
esbuild copies the package from node_modules, so we need to be sure that the data is updated before the build.